src/kontrol/prove.py

```diff
@@ -345,7 +346,11 @@ def create_kcfg_explore() -> KCFGExplore:
 
             if proof is None:
                 # With CSE, top-level proof should be a summary if it's not a test or setUp function
-                if options.cse and options.config_type == ConfigType.TEST_CONFIG and not test.contract.is_test_contract:
+                if (
+                    (options.cse or options.include_summaries)
+                    and options.config_type == ConfigType.TEST_CONFIG
+                    and not test.contract.is_test_contract
+                ):
```

Comment:
Accounts should be structured whenever either `--cse` or `--include-summary` is used.